<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MarkDialog</class>
 <widget class="QWidget" name="MarkDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>466</width>
    <height>294</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add Mark</string>
  </property>
  <property name="toolTip">
   <string/>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0" colspan="4">
    <widget class="QComboBox" name="cBMarkerName">
     <property name="toolTip">
      <string/>
     </property>
     <property name="editable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QDoubleSpinBox" name="sBLatitude">
     <property name="suffix">
      <string>°</string>
     </property>
     <property name="decimals">
      <number>9</number>
     </property>
     <property name="minimum">
      <double>-180.000000000000000</double>
     </property>
     <property name="maximum">
      <double>180.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="2">
    <widget class="QLabel" name="lblPosition">
     <property name="text">
      <string>Latitutde / Longitude / Height</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QPushButton" name="btnKeyDlg">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="icon">
      <iconset resource="../icon/resources.qrc">
       <normaloff>:/buttons/key</normaloff>:/buttons/key</iconset>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="4">
    <widget class="QLineEdit" name="lEMarkerComment">
     <property name="toolTip">
      <string/>
     </property>
    </widget>
   </item>
   <item row="7" column="2">
    <widget class="QDoubleSpinBox" name="sBHeight">
     <property name="suffix">
      <string> m</string>
     </property>
     <property name="decimals">
      <number>3</number>
     </property>
     <property name="minimum">
      <double>-500.000000000000000</double>
     </property>
     <property name="maximum">
      <double>9000.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="7" column="3">
    <widget class="QPushButton" name="btnPosition">
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="lblComment">
     <property name="text">
      <string>Comment</string>
     </property>
    </widget>
   </item>
   <item row="8" column="0" colspan="4">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="QDoubleSpinBox" name="sBLongitude">
     <property name="suffix">
      <string>°</string>
     </property>
     <property name="decimals">
      <number>9</number>
     </property>
     <property name="minimum">
      <double>-90.000000000000000</double>
     </property>
     <property name="maximum">
      <double>90.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QCheckBox" name="cBMarkerNameC">
     <property name="toolTip">
      <string/>
     </property>
     <property name="text">
      <string>Marker Name</string>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QLabel" name="lblMarker">
     <property name="text">
      <string>%r=001</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="4">
    <widget class="QGroupBox" name="gBPositionMode">
     <property name="title">
      <string>Position Mode</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QRadioButton" name="rBGo">
        <property name="toolTip">
         <string/>
        </property>
        <property name="text">
         <string>&amp;GO (Kinematic)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rBStop">
        <property name="toolTip">
         <string/>
        </property>
        <property name="text">
         <string>STOP (Stati&amp;c)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rBFix">
        <property name="text">
         <string>FIX (Fixed)</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>cBMarkerNameC</tabstop>
  <tabstop>cBMarkerName</tabstop>
  <tabstop>lEMarkerComment</tabstop>
 </tabstops>
 <resources>
  <include location="../icon/resources.qrc"/>
 </resources>
 <connections/>
</ui>
